protocol/x/subaccounts/keeper/subaccount.go

```diff
@@ -280,7 +280,10 @@ func (k Keeper) UpdateSubaccounts(
 		// Emit an event indicating a funding payment was paid / received for each settled funding
 		// payment. Note that `fundingPaid` is positive if the subaccount paid funding,
 		// and negative if the subaccount received funding.
-		for perpetualId, fundingPaid := range fundingPayments {
+		// Note the perpetual IDs are sorted first to ensure event emission determinism.
+		sortedPerpsWithFunding := lib.GetSortedKeys[lib.Sortable[uint32]](fundingPayments)
```

Comment:
nit: `sortedPerpIds` as the variable name?

Reply:
Updated!